draft-ietf-teep-protocol.md

```diff
@@ -96,6 +96,8 @@ informative:
   I-D.ietf-teep-architecture: 
   I-D.lundblade-rats-eat-media-type:
   I-D.ietf-rats-concise-ta-stores:
+  I-D.lundblade-rats-eat-media-type:
+  I-D.wallace-rats-concise-ta-stores:
```

Comment:
Wrong reference, this was obsoleted by line 98.  Change anything that uses this to use that one.